tests/test_python-opensubtitles.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+from os import path
+
+
+class TestPythonOpenSubtitles(object):
+    @staticmethod
+    @pytest.fixture
+    def data():
+        class Data:
+            username = 'doctest'
+            password = 'doctest'
+            name = 'Dark City'
+            path = '/home/alex/Videos/Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo/'
+            video = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.mp4'
+            subtitle = 'Dark.City.1998.Directors.Cut.BRRip.H264.AAC.5.1ch.Gopo.srt'
+        return Data
+
+    @staticmethod
+    @pytest.fixture
+    def os():
+        from pythonopensubtitles.opensubtitles import OpenSubtitles
+        return OpenSubtitles()
+
+    @staticmethod
+    @pytest.fixture
+    def file(data):
+        from pythonopensubtitles.utils import File
+        return File(path.join(data.path, data.video))
```

Comment:
I still don-t understand how this is going to work if the file doesn't exist. I couldn-t see the mock anywhere and call something like `file.get_hash()` will probably return an `IOError`. What am I missing? 🤔